roles/confluent.variables/defaults/main.yml

```diff
@@ -1101,6 +1106,314 @@ control_center_health_check_user: "{{control_center_ldap_user}}"
+kafka_connect_replicator_sasl_scram_principal: "{{ sasl_scram_users.kafka_connect_replicator.principal }}"
+
+### SCRAM password for Kafka Connect Replicator to authenticate with.
+kafka_connect_replicator_sasl_scram_password: "{{ sasl_scram_users.kafka_connect_replicator.password }}"
+
+### SASL PLAIN principal for Kafka Connect Replicator to authenticate with.
+kafka_connect_replicator_sasl_plain_principal: "{{ sasl_plain_users.kafka_connect_replicator.principal }}"
+
+### SASL PLAIN password for Kafka Connect Replicator to authenticate with.
+kafka_connect_replicator_sasl_plain_password: "{{ sasl_plain_users.kafka_connect_replicator.password }}"
+
+### Boolean that defines if the Jolokia agent is enabled on Kafka Connect Replicator.
+kafka_connect_replicator_jolokia_enabled: "{{jolokia_enabled}}"
+
+### Sets the auth mode for the Jolokia Agent on Kafka Connect Replicator.
+kafka_connect_replicator_jolokia_auth_mode: "{{jolokia_auth_mode}}"
+
+### Username for Jolokia Agent when using Basic Auth.
+kafka_connect_replicator_jolokia_user: "{{jolokia_user}}"
+
+### Password for Jolokia Agent when using Basic Auth.
+kafka_connect_replicator_jolokia_password: "{{jolokia_password}}"
+
+### Port for Jolokia agent for Kafka Connect Replicator.
+kafka_connect_replicator_jolokia_port: 7777
+
+kafka_connect_replicator_jolokia_config: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/confluent-replicator-jolokia.properties"
+
+### Full path to download the Jolokia Agent Jar.
+kafka_connect_replicator_jolokia_jar_path: /opt/jolokia/jolokia.jar
+
+### Boolean to enable Prometheus Exporter Agent installation and configuration on Kafka Connect Replicator.
+kafka_connect_replicator_jmxexporter_enabled: "{{jmxexporter_enabled}}"
+
+### Port to expose prometheus metrics. Beware of port collisions if colocating components on same host.
+kafka_connect_replicator_jmxexporter_port: 8071
+
+kafka_connect_replicator_jmxexporter_config_path: /opt/prometheus/kafka_connect_replicator.yml
+
+### Set this variable to customize the directory that Kafka Connect Replicator writes log files to. Default location is /var/log/kafka.
+kafka_connect_replicator_log_dir: "{{kafka_connect_replicator.appender_log_path}}"
+
+### Set this variable to customize the default log name that Kafka Connect Replicator writes logs to. Default name is kafka-connect-replicator.log.
+kafka_connect_replicator_default_log_name: "{{kafka_connect_replicator.appender_log_name}}"
+
+### Set this variable to customize the default max number of log files generated by Kafka Connect Replicator.  Default is 10.
+kafka_connect_replicator_max_log_files: "{{kafka_connect_replicator.appender_max_log_files}}"
+
+### Set this variable to customize the default max size of a log file generated by Kafka Connect Replicator.  Default is 100mb.
+kafka_connect_replicator_log_file_size: "{{kafka_connect_replicator.appender_log_file_size}}"
```

Comment:
for the other components these fields are no longer customizable in this branch

Reply:
and the log name as well I think

Reply:
Why is that?  To me it seems like we are removing previously available functionality.  Customers will want to customize there logging, it's a pretty regular occurrence actually, as they will want to repoint where logs are written to and how much they keep.  For example, some customers will need to keep 30 days worth of logs for compliance reasons.

Reply:
My reasoning is for components like c3, RP, and SR. We use out of the box log4j files shipped with the packages and thus do not allow for customizing those settings within their files, so i removed that functionality across the rest of the components in this PR: https://github.com/confluentinc/cp-ansible/pull/411

Even ksql youll see is hardcoded: https://github.com/confluentinc/cp-ansible/blob/6.1.x/roles/confluent.ksql/templates/ksql-server_log4j.properties.j2#L43

We should mark it in jira if you think this is a big problem, but that is the current standard that made it into the 61x branch.

Reply:
Yep, and it's not correct unfortunately, to the point that is should be flagged as major bug.  This will literally break use cases for customers.  I am going to file a bug for it now.

Reply:
I made sure to keep all these customizable vars in the replicator-dom branch. this can be marked resolved